db/migrate/20160707112342_remove_banking_info_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveBankingInfoFromUsers < ActiveRecord::Migration
+  def change
+  	remove_column :users, :banking_info, :text
```

Comment:
Purely cosmetic stuff: the convention for indenting Ruby code is 2 spaces, no tabs. Your text editor can be set to enforce that, but the configuration options depend on the editor.